src/main/java/page_objects/BookTicketPage.java

```diff
@@ -0,0 +1,48 @@
+package page_objects;
+
+import helper.BrowserHelper;
+import helper.Constant;
+import helper.ElementHelper;
+import helper.element_helper.Dropdown;
+import helper.element_helper.Label;
+import models.Ticket;
+import org.openqa.selenium.By;
+
+public class BookTicketPage extends GeneralPage {
+
+    //Locator
+    private final Dropdown ddlDepartDate = new Dropdown(By.cssSelector("[name=Date]"));
+    private final Dropdown ddlDepartFrom = new Dropdown(By.cssSelector("[name=DepartStation]"));
+    private final Dropdown ddlArriveAt = new Dropdown(By.cssSelector("[name=ArriveStation]"));
+    private final Dropdown ddlSeatType = new Dropdown(By.cssSelector("[name=SeatType]"));
+    private final Dropdown ddlTicketAmount = new Dropdown(By.cssSelector("[name=TicketAmount]"));
+    private final Dropdown btnSubmitBookTicket = new Dropdown(By.cssSelector("input[value='Book ticket']"));
+    private final Label lblErrorMessage = new Label(By.cssSelector(".message.error"));
+    private final Label lblErrorMessageAmount = new Label(By.cssSelector(".validation-error"));
```

Comment:
again, Vietnamese to English

Reply:
Sorry i  repeat error again ,  i will overcome it now